src/app/find-help/accommodation-search/accommodation-search-form/accommodation-search-form.component.ts

```diff
@@ -43,4 +42,15 @@ export class AccommodationSearchFormComponent implements OnInit {
     await this.storeUrlService.setCustomPaginatorParam(param);
     this.search.emit(this.data);
   }
+
+  async clearFilters(): Promise<void> {
+    this.data = {};
+    const param: Params = {
+      page: 0,
+      size: localStorage.getItem(LocalStorageKeys.PageSize) ?? 5,
```

Comment:
Jesteś pewien, że to też chcemy czyścić, a nie tylko filtry z górnej części strony?
Skonsultuj proszę najlepiej z Dominiką

Reply:
Propertiesów page, size i sort już nie czyszczę, natomiast odświeżam jeszcze pozostałe queryParams, ponieważ nie były one usuwane z urla, przez co po wyczyszczeniu filtrów i odświeżeniu strony wskakiwały wcześniejsze paramsy.